src/Microsoft.AspNet.Razor/RazorResources.resx

```diff
@@ -419,4 +419,10 @@ Instead, wrap the contents of the block in "{{}}":
   <data name="TagHelperParseTreeRewriter_EndTagTagHelperMustNotHaveAnEndTag" xml:space="preserve">
     <value>Found an end tag (&amp;lt;/{0}&amp;gt;) for tag helper '{1}' with tag structure that disallows an end tag ('{2}').</value>
   </data>
+  <data name="TagHelperParseTreeRewriter_CannotHaveNonTagContent" xml:space="preserve">
+    <value>The parent '&amp;lt;{0}&amp;gt;' tag helper does not allow non-tag content. Only child tag helper(s) targeting tag name(s) '{1}' are allowed.</value>
+  </data>
+  <data name="TagHelperParseTreeRewriter_InvalidNestedTag" xml:space="preserve">
+    <value>The &amp;lt;'{0}'&amp;gt; tag is not allowed by parent '&amp;lt;{1}&amp;gt;' tag helper. Only child tag helper(s) targeting tag name(s) '{2}' are allowed.</value>
```

Comment:
Single quotes are not useful mixed with angle brackets. Expect `<disallowed> tag` here and `parent <allower> tag helper` here and above.